Updating getting started to include the minimal sample for .NET 6 (#1287)
microsoft/reverse-proxy: docs/docfx/articles/getting-started.md

YARP is implemented on top of .NET Core infrastructure and is usable on Windows, Linux or MacOS. Development can be done with the SDK and your favorite editor, [Microsoft Visual Studio](https://visualstudio.microsoft.com/vs/) or [Visual Studio Code](https://code.visualstudio.com/).

> YARP 1.0.0 Preview 12 supports ASP.NET Core 3.1, 5.0. & 6.0. You can download the .NET SDK from https://dotnet.microsoft.com/download/dotnet/

Reviewer: ```suggestion
YARP 1.0.0 Preview 12 supports ASP.NET Core 3.1, 5.0 & 6.0. You can download the .NET SDK from https://dotnet.microsoft.com/download/dotnet/
```

} 
```
 
## .NET 6 support

> In addition to supporting the style of startup used by .NET 5, .NET 6 introduces the ability to have [top level statements](https://docs.microsoft.com/en-us/dotnet/csharp/fundamentals/program-structure/top-level-statements) in your app. This combined with [additional glue in ASP.NET](https://devblogs.microsoft.com/aspnet/asp-net-core-updates-in-net-6-preview-4/#introducing-minimal-apis) has signifcantly reduced the complexity of the basic ASP.NET project template, and the additions for YARP.

Reviewer 2: ```suggestion
In addition to supporting the style of startup used by .NET 5, .NET 6 introduces the ability to have [top level statements](https://docs.microsoft.com/dotnet/csharp/fundamentals/program-structure/top-level-statements) in your app. This combined with [additional glue in ASP.NET](https://devblogs.microsoft.com/aspnet/asp-net-core-updates-in-net-6-preview-4/#introducing-minimal-apis) has signifcantly reduced the complexity of the basic ASP.NET project template, and the additions for YARP.
```

> In addition to supporting the style of startup used by .NET 5, .NET 6 introduces the ability to have [top level statements](https://docs.microsoft.com/en-us/dotnet/csharp/fundamentals/program-structure/top-level-statements) in your app. This combined with [additional glue in ASP.NET](https://devblogs.microsoft.com/aspnet/asp-net-core-updates-in-net-6-preview-4/#introducing-minimal-apis) has signifcantly reduced the complexity of the basic ASP.NET project template, and the additions for YARP.

Reviewer 3: ```suggestion
In addition to supporting the style of startup used by .NET 5, .NET 6 introduces the ability to have [top level statements](https://docs.microsoft.com/en-us/dotnet/csharp/fundamentals/program-structure/top-level-statements) in your app. This combined with [additional glue in ASP.NET](https://devblogs.microsoft.com/aspnet/asp-net-core-updates-in-net-6-preview-4/#introducing-minimal-apis) has significantly reduced the code for a basic ASP.NET project template, and the additions for YARP.
```
